README.md

```diff
@@ -176,6 +202,34 @@ $channel = $bunny->channel();
 $channel->queueDeclare('queue_name'); // Queue name
 ```
 
+<details>
+  <summary>Full Example</summary>
+
+  ```php
+  use Bunny\Client;
+  use Bunny\Configuration;
+  use React\EventLoop\Loop;
+  
+  use function React\Async\async;
+  
+  $configuration = new Configuration(
+      host:     'HOSTNAME',
+      vhost:    'VHOST',    // The default vhost is /
+      user:     'USERNAME', // The default user is guest
+      password: 'PASSWORD', // The default password is guest
+  );
+  
+  $bunny = new Client($configuration);
+  Loop::futureTick(async(static function (): void {
```

Comment:
Why is it necessary to use `Loop::futureTick`, but not `async`? I'm not sure whether it is a necessary detail to be known by the developer for classic use cases (php-fpm with no full async application)

Reply:
> Why is it necessary to use Loop::futureTick, but not async?
`Loop::futureTick` isn't strictly necessary, `async` is tho.

> I'm not sure whether it is a necessary detail to be known by the developer for classic use cases (php-fpm with no full async application)
It shouldn't matter to them. Will have a look at adding a section for that tomorrow.